src/python_bindings/bindings.cpp

```diff
@@ -185,7 +185,14 @@ PYBIND11_MODULE(VecSim, m) {
         .def_readwrite("hnswParams", &VecSimParams::hnswParams)
         .def_readwrite("bfParams", &VecSimParams::bfParams);
 
-    py::class_<VecSimQueryParams>(m, "VecSimQueryParams").def(py::init());
+    py::class_<VecSimQueryParams> queryParams(m, "VecSimQueryParams");
+
+    queryParams.def(py::init<>())
+        .def_readwrite("hnswRuntimeParams", &VecSimQueryParams::hnswRuntimeParams);
+
+    py::class_<HNSWRuntimeParams>(queryParams, "HNSWRuntimeParams")
+        .def(py::init<>())
+        .def_readwrite("efRuntime", &HNSWRuntimeParams::efRuntime);
```

Comment:
what about other members of the query parameters?

Reply:
They are irrelevant, as the hybrid policy and the batch size are parameters in the hybrid iterator level of`rediSearch`, and they don't mean anything in the level of the "lower" batch iterator API...